schedule/yast/msdos.yaml

```diff
@@ -20,6 +23,7 @@ schedule:
   - installation/start_install
   - installation/await_install
   - '{{installation/logs_from_installation_system}}'
+  - installation/teardown_libyui
```

Comment:
We should schedule it after `installation/reboot_after_installation`, this is also the reason why your VR has failed.